YubiKit/YubiKit/Sessions/Shared/Services/OATH/YKFKeyOATHService.m

```diff
@@ -112,6 +112,21 @@ - (void)executeDeleteRequest:(YKFKeyOATHDeleteRequest *)request completion:(YKFK
     }];
 }
 
+- (void)executeRenameRequest:(YKFKeyOATHRenameRequest *)request completion:(YKFKeyOATHServiceCompletionBlock)completion {
+    YKFParameterAssertReturn(request);
+    YKFParameterAssertReturn(completion);
+    
+    YKFKeySessionError *credentialError = [YKFOATHCredentialValidator validateCredential:request.credential includeSecret:YES];
```

Comment:
when you rename credential, you don't have a secret (it's always null unless you set it for account creation method).
Potentially you can have null pointer exception